src/types/index.ts

```diff
@@ -31,6 +31,13 @@ export type ObjectValues<T> = T[keyof T];
  */
 type PublicPart<T> = {[K in keyof T]: T[K]};
 
+/**
+ * Allowed options that can be passed to our fetch utility functions
+ */
+export interface IRequestInitBodyParsed extends Omit<RequestInit, 'body'> {
+  body?: object | null;
```

Comment:
Do we need `undefined` and `null` both here?

Reply:
This is overriding native browser interface. But you have right: for our conveniency we can remove the `null`.
Done